exercises/word-count/word-count.spec.js

```diff
@@ -29,12 +29,12 @@ describe('words()', () => {
   });
 
   xit('respects case', () => {
```

Comment:
Same as xjavascript :smile:
I think this description does not reflect what the spec does now. What about normalizes to lower case? @matthewmorgan, if you agree, I can do the change while merging the PR.